.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  extends: "steelbrain"
```

Comment:
by the way, if it's not installer and you try to lint with eslint, it cries out loud that `eslint-config-steelbrain` is missing :wink: